### PR TITLE
Added my information to the about me page

### PR DESCRIPTION
**Pull request by miaboloix**

### CODE REVIEW

**Review by niccireinking — APPROVED** [review]
lgtm

File: src/main/webapp/aboutus.html [hunk 1]
```diff
@@ -30,11 +30,11 @@
       </ul>
     </nav>
     <h1>About Our Team</h1>
-    <h2>Teammate A Name</h2>
+    <h2>Mia Boloix</h2>
     <ul>
-      <li>Summer Feelz: </li>
-      <li>Aspirational Hobby: </li>
-      <li>Ask me About: </li>
+        <li>Summer Feelz: Very excited, a little overwhelmed, and so ready!!</li>
```
Comment (emto) [inline src/main/webapp/aboutus.html:35]: Fix indentation -- lets discuss in the chat whether we want 2 spaces or 4 spaces.

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -33,7 +33,7 @@ <h1>CodeU Starter Project</h1>
     <p>This is the CodeU starter project. Click the links above to login and visit your page.
     You can post messages on your page, and you can visit other user pages if you have
     their URL.</p>
-    <p>This is your code now! Feel free to make changes, and don't be afraid to get creative!
-    You could start by modifying this page to tell the world more about your team.</p>
```
Comment (emto) [inline src/main/webapp/index.html:37]: intentional change?

Comment (miaboloix) [reply src/main/webapp/index.html:37]: This was me trying to figure out what they wanted for index.html changes for week0... should i change this back?

Comment (emto) [reply src/main/webapp/index.html:37]: Yeah lets keep the info on the about us page

File: src/main/webapp/aboutus.html [hunk 3]
```diff
@@ -30,11 +30,11 @@
       </ul>
     </nav>
     <h1>About Our Team</h1>
-    <h2>Teammate A Name</h2>
+    <h2>Mia Boloix</h2>
     <ul>
-      <li>Summer Feelz: </li>
-      <li>Aspirational Hobby: </li>
-      <li>Ask me About: </li>
+         <li>Summer Feelz: Very excited, a little overwhelmed, and so ready!!</li>
```
Comment (emto) [inline src/main/webapp/aboutus.html]: I think the indents right now are 5 spaces in this change. let's go back to 2 spaces and do a mass indent fix later.

**Review by emto — APPROVED** [review]

**Pull request merged**